docs/contributing/rdg-storage-format.rst

```diff
@@ -0,0 +1,61 @@
+=========================
+Katana RDG Storage Format
+=========================
+
+Katana persists its graphs on disk in a format called `RDG`. This `RDG` is a directory filled with files that define the graph.
```

Comment:
nit: define what the acronym for RDG stands for

Reply:
someone would have to explain it to me first hehehe :)

Reply:
RDG: Resilient Distributed Graph, a tribute to Spark's Resilient Distributed Dataset (RDD).

Reply:
Thanks! Added